README.md

```diff
@@ -38,3 +50,7 @@ are:
    ```
    docker run --rm --entrypoint /usr/bin/env fpco/pid1 /sbin/pid1 ps
    ```
+   
+If running pid1 standalone outside of docker, pid1 supports similar run command
```

Comment:
I was thinking similar in terms of the `-u`, `-g`, `-e`, and `-w` pid1 command line options being the same as supported by `docker run`, but I see how that statement can be confusing. I'll just delete that paragraph, lines 54-56, as it doesn't really provide any more information than the usage description.